hnn_core/tests/test_network.py

```diff
@@ -956,3 +838,186 @@ def test_network_mesh():
     _ = jones_2009_model(mesh_shape=mesh_shape)
     _ = calcium_model(mesh_shape=mesh_shape)
     _ = law_2021_model(mesh_shape=mesh_shape)
+
+
+class TestPickConnection:
+    """Tests for the pick_connection function."""
+    @pytest.mark.parametrize("arg_name",
+                             ["src_gids", "target_gids", "loc", "receptor"]
+                             )
+    def test_1argument_none(self, base_network, arg_name):
+        """ Tests passing None as an argument value. """
+        net, _ = base_network
+        kwargs = {'net': net, f'{arg_name}': None}
+        indices = pick_connection(**kwargs)
+        assert not indices
+
+    @pytest.mark.parametrize("arg_name", ["src_gids", "target_gids"])
+    def test_1argument_range(self, base_network, arg_name):
+        """ Tests passing range as an argument value. """
```

Comment:
```suggestion
        """Tests passing range as an argument value."""
```

nitpicky but I think it's pep 257 requirement ... must be one line with no spaces ... please fix in the remaining document too